lib/opencl/kernels/kv_corr_sm_legacy.cl

```diff
@@ -0,0 +1,113 @@
+//#define SAMPLES_PER_DATA_SET
+//#define NUM_ELEMENTS
+//#define NUM_FREQS
+//#define BLOCK_SIZE
+//#define COARSE_BLOCK_SIZE
```

Comment:
These are commented out here but are actually used in the code.

Reply:
Ah those are passed in at run-time.  I'll add a note to that effect.